cl/citations/tests.py

```diff
@@ -761,3 +792,113 @@ def test_making_edge_list(self) -> None:
                 a=a,
             ):
                 self.assertEqual(make_edge_list(q), a)
+
+
+class FilterParentheticalTest(SimpleTestCase):
+    def test_is_descriptive(self):
+        test_pairs = [
+            ("holding that 2 + 2 = 5", True),
+            (
+                "accountant who gave lay opinion testimony might have qualified as expert",
+                True,
+            ),
+            (
+                "where plaintif's complaint alleges facts which, if proven, would entitle plaintiff to relief under the Eighth Amendment, dismissal of complaint was inappropriate",
+                True,
+            ),
+            (
+                "ruling that there is nothing either legal or illegal, only thinking makes it so",
+                True,
+            ),
+            ("First Amendment", True),
+            ("mislabeled product", True),
+            ('"Look on my Works, ye Mighty, and despair"', True),
+            (
+                '"Texas does not seek to have the Court interpret the Constitution, so much as disregard it."',
+                True,
+            ),
+            ("questioning whether he who made the Lamb made thee", True),
+            ("Section 403(d)(2)", True),
```

Comment:
All of these register to me as ones that should be False, right?

Reply:
This gets at a philosophical decision as to what we consider useful. 

The approach we took was to try to filter out what we considered citation "metadata" (or "spam", when we talked about it) while preserving as much as possible anything "substantive". While something like "First Amendment" or "mislabeled product" is not very useful to show on the opinion page on CourtListener (and would be downranked by our heuristic in #1906), we thought it could still potentially be of interest to a researcher were these parentheticals to be published in some kind of bulk dataset.

That's the reason -- to address a couple of your other comments -- we took a pretty fine-grained approach to filtering and didn't place a blanket length restriction. I can definitely see though how this increases complexity and maintenance burden, though, so I'm open to whatever you think is the best course of action.

Reply:
Gotcha. I'll add another argument to your side: If we gather everything in the DB, we can more easily see what we have and refine from there. You've convinced me here.